extensions-contrib/kubernetes-overlord-extensions/README.md

```diff
@@ -0,0 +1,222 @@
+            readOnly: true
+          - mountPath: /druid/data
+            name: data-volume
+          - mountPath: /druid/deepstorage
+            name: deepstorage-volume
+        restartPolicy: "Never"
+        securityContext:
+          fsGroup: 1000
+          runAsGroup: 1000
+          runAsUser: 1000
+        tolerations:
+        - effect: NoExecute
+          key: node.kubernetes.io/not-ready
+          operator: Exists
+          tolerationSeconds: 300
+        - effect: NoExecute
+          key: node.kubernetes.io/unreachable
+          operator: Exists
+          tolerationSeconds: 300
+        volumes:
+        - configMap:
+            defaultMode: 420
+            name: druid-tiny-cluster-peons-config
+          name: nodetype-config-volume
+        - emptyDir: {}
+          name: data-volume
+        - emptyDir: {}
+          name: deepstorage-volume
+    ```
+    
+    Runtime properties that need to be passed to the Job's peon process
+    
+    ```
+    druid.port=8100 (what port the peon should run on)
+    druid.peon.mode=remote
+    druid.service=druid/peon (for metrics reporting)
+    druid.indexer.task.baseTaskDir=/druid/data (this should match the argument to the ./peon.sh run command in the PodTemplate)
+    druid.indexer.runner.type=k8s
+    druid.indexer.task.encapsulatedTask=true
+    ```
+
+    Any runtime property or jvm config used by the peon process can also be passed here
+
+    E.G. below is a example of a ConfigMap that can be used to generate the nodetype-config-volume mount in the above template.
+    ```
+    kind: ConfigMap
+    metadata:
+    name: druid-tiny-cluster-peons-config
+    namespace: default
+    apiVersion: v1
+    data:
+        jvm.config: |-
+            -server
+            -XX:MaxDirectMemorySize=1000M
+            -Duser.timezone=UTC
+            -Dfile.encoding=UTF-8
+            -Dlog4j.debug
+            -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager
+            -Djava.io.tmpdir=/druid/data
+            -Xmx1024M
+            -Xms1024M
+        log4j2.xml: |-
+            <?xml version="1.0" encoding="UTF-8" ?>
+            <Configuration status="WARN">
+                <Appenders>
+                    <Console name="Console" target="SYSTEM_OUT">
+                        <PatternLayout pattern="%d{ISO8601} %p [%t] %c - %m%n"/>
+                    </Console>
+                </Appenders>
+                <Loggers>
+                    <Root level="info">
+                        <AppenderRef ref="Console"/>
+                    </Root>
+                </Loggers>
+            </Configuration>
+        runtime.properties: |
+            druid.port=8100
+            druid.service=druid/peon
+            druid.server.http.numThreads=5
+            druid.indexer.task.baseTaskDir=/druid/data
+            druid.indexer.runner.type=k8s
+            druid.peon.mode=remote
+            druid.indexer.task.encapsulatedTask=true
+    ```
+
```

Comment:
Thank you for adding these examples. I have not gone through them in detail yet, but there is another doc in docs/development/extensions-contrib/k8s-jobs.md that appears to attempt to answer similar things.

I think adding the examples to the linked page will be helpful, as they are published with the docs.

The README is also helpful as people will come to this file when they hear about the extension, so you could have the README just link out to the other page.

Reply:
i didn't notice that file, will take a look